src/Renderer.php

```diff
@@ -76,8 +76,16 @@ public static function attributes(string ...$attributes): string
             if (is_array($value)) {
                 $value = implode(' ', $value);
             }
-            $value = htmlspecialchars($value);
-            $attrs .= " {$key}='{$value}'";
+
+            if( $value === true )
+            {
+                $attrs .= " {$key}='{$value}'";
```

Comment:
Boolean attributes don't care about the value passed ie 'boolean-attr="false"' still registers as true since it is present so the true case should just append the key

Reply:
You had it right in your issue, is there any specific reason you changed it ?